crates/ruff_cli/src/commands/rule.rs

```diff
@@ -32,13 +32,18 @@ pub fn rule(rule: Rule, format: HelpFormat) -> Result<()> {
             output.push('\n');
             output.push('\n');
 
-            if let Some(autofix) = rule.autofixable() {
-                output.push_str(match autofix.available {
-                    Availability::Sometimes => "Autofix is sometimes available.",
-                    Availability::Always => "Autofix is always available.",
-                });
-                output.push('\n');
-                output.push('\n');
+            match rule.autofixable() {
+                AutofixKind::Sometimes => {
+                    output.push_str("Autofix is sometimes available.");
+                    output.push('\n');
+                    output.push('\n');
+                }
+                AutofixKind::Always => {
+                    output.push_str("Autofix is always available.");
+                    output.push('\n');
+                    output.push('\n');
+                }
+                AutofixKind::None => {}
             }
```

Comment:
Nit: We could implement `Display` for `AutofixKind`. You could then write this as
```suggestion
		let autofix = rule.autofixable();
		if matches!(autofix, AutofxKind::Sometimes | AutofixKind::Always) {
			writeln!(output, "{autofix}\n").unwrap(); // Writting to a string never fails except if the program OOms
		}
```

and then re-use the same logic in `generate_docs`

Reply:
I do not know the term `OOms`. What does it mean?

Reply:
Oh, sorry, That's my java background shining through. OOM is an acronym for Out of Memory (exception in Java). And *OOms* is the activity when the program runs out of memory.